### DISCUSSION TITLE
Multiple full nodes

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#6128] [Support]
If i have multiple full nodes farming/plotting, is there a chance that I wont get a reward due to conflict or something?

Is there a chance that the network only sees the node that has the less plots for example and the rest are ignored?

How can I check that everything is working ok?

Is it better to have the following?
1 full node that has mapped all other machine's drives.
The rest of the nodes that plot just add plots on hdds that are shared in the private network and are are mapped by the single full node.

**Comment by BrandtH22** [comment]
@GeorgePapageorgakis , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - detailed instructions for farming on multiple machines can be found here https://docs.chia.net/farming-on-many-machines

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
